### DISCUSSION TITLE
CORS ERROR 발생

### DISCUSSION THREAD

**Discussion by qksdhrehf** [changdoc/naver-vaccine-macro#240] [Q&A]
항상 감사드립니다!
다름이 아니라 네이버 플레이스 상에서 프로그램을 돌리다보면
잘돌아가다가 어느순간부터 [자동새로고침종료] 버튼 옆에 있는 새로고침은 돌아가지만, 시간이 최신으로 반영되지 않아
개발자도구를 켜서 graphql 의 status를 확인했는데, 정상적이었다면 200으로 출력되었을것이 CORS 에러가 난다고 표기가 되었습니다.
이를 해결하기 위해 다른 브라우저를 사용하기도 하고, 크롬 아이콘 주소에 --disable-web-security --user-data-dir="C:\chrome" 를 붙여 CORS 기능을 꺼보기도 했습니다만, 처음에는 되다가 다시 에러가 나기 시작했습니다.
혹시 이와 관련해서 해결방법이 있을까요?

감사합니다.

**Comment by changdoc** [comment]
여러분들이 CORS에러로 문의를 해주셨는데 저는 재현을 못하고 있어 해결 방법을 찾지 못했습니다.
지속적으로 발생한다면 카카오맵에서 매크로 동작 시키는게 좋을 것 같네요.

**Reply by hjpae** [reply]
매크로 자체 문제라기보다는 네이버지도 측에서 잔여백신 업데이트를 불러올 때 발생하는 에러인 것으로 생각됩니다.
